### ISSUE TITLE
Regression: jupyter_client 7.02 broke lastest jupyter_console

### ISSUE THREAD

**Issue by slonik-az** [jupyter/jupyter_client#690]
The regression was first reported to the `jupyter_console` repo https://github.com/jupyter/jupyter_console/issues/251

Cross-posting here because `jupyter_client` changes from 6.x to 7.x seem to be the root cause.

**Comment by davidbrochart** [comment]
I didn't look at https://github.com/jupyter/jupyter_console/issues/251 yet, but I remember I opened https://github.com/jupyter/jupyter_console/pull/244.

**Comment by slonik-az** [comment]
Yes, I think the issues raised in https://github.com/jupyter/jupyter_console/pull/244 (still open and pending resolution) are the root cause of the regression. Downgrading `jupyter_client` from 7.0.2 to  6.1.12 resolved the `jupyter console` problem for me.
